src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/emptyblock/InputEmptyBlockSemanticText.java

```diff
@@ -0,0 +1,86 @@
+package com.puppycrawl.tools.checkstyle.checks.blocks.emptyblock;
+
+import java.io.*; // star import for instantiation tests
+import java.awt.Dimension; // explicit import for instantiation tests
+import java.awt.Color;
+
+/* Config:
+ * option = "text"
+ * tokens = "LITERAL_WHILE, LITERAL_TRY, LITERAL_FINALLY, LITERAL_DO, LITERAL_IF, LITERAL_ELSE,
+ *           LITERAL_FOR, INSTANCE_INIT, STATIC_INIT, LITERAL_SWITCH, LITERAL_SYNCHRONIZED"
+ */
+/**
+ * Test case for detecting empty block statements.
+ **/
+class InputEmptyBlockSemanticText
+{
+    static {
+        Boolean x = new Boolean(true);
+    }
+
+    {
+        Boolean x = new Boolean(true);
+        Boolean[] y = new Boolean[]{Boolean.TRUE, Boolean.FALSE};
+    }
+
+    Boolean getBoolean()
+    {
+        return new java.lang.Boolean(true);
+    }
+
+    void exHandlerTest()
+    {
+        try {
```

Comment:
Placing `// violation` comment here would result in a comment being included in empty `try` block, which would then **not** produce the violation.
However, if we do it like this - 

```java
try {} // violation
```
Then we can get around the issue. But we introduce another issue - code coverage. Using this method results in code coverage ratio dropping from 1 to 0.97, which is unacceptable. 
Please help me get around this problem.

Reply:
Why not just do:
```java
    try {
    }
// ^ violation
```

We only check to make sure that one `ok` or `violation` comment exists in a file, and that `* Config` comment exists. Except for Indentation, Check UT's do not check line numbers.

Reply:
>Except for Indentation, Check UT's do not check line numbers.

That is very helpful to know, now the fix is easy. Thanks!